includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -125,6 +151,43 @@ public function get_all_readers( $request ) {
 		}
 	}
 
+	/**
+	 * Links a card reader to an account and terminal location.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function register_reader( $request ) {
+		try {
+			$response = $this->api_client->register_terminal_reader(
+				$request->get_param( 'location' ),
+				$request->get_param( 'registration_code' ),
+				$request->get_param( 'label' ),
+				$request->get_param( 'metadata' )
+			);
+
+			$reader = [
```

Comment:
There is a [function in WordPress](https://developer.wordpress.org/reference/functions/wp_array_slice_assoc/) just for this:

```php
$reader = wp_array_slice_assoc( $response, [ 'id', 'livemode', 'device_type', 'label', 'location', 'metadata', 'status' ] );
```

Reply:
Fixed in 3fee0e7 👍